chromadb/api/types.py

```diff
@@ -135,6 +112,43 @@ class IncludeEnum(str, Enum):
     data = "data"
 
 
+# Record set
+class RecordSet(TypedDict):
+    ids: IDs
+    embeddings: Optional[Embeddings]
+    metadatas: Optional[Metadatas]
+    documents: Optional[Documents]
+    images: Optional[Images]
+    uris: Optional[URIs]
+
+
+def does_record_set_contain_data(record_set: RecordSet, include: List[str]) -> bool:
```

Comment:
Maybe an empty `include` here could mean that any of the fiends in `RecordSet` is populated.

Reply:
Could we use `IncludeEnum` here?

Reply:
Not an exact fit since IncludeEnum does not have `images`. RecordSet also does not have a `data` field